Core/src/IfcGeometryConverter/GeomUtils.cpp

```diff
@@ -432,6 +395,53 @@ void GeomUtils::extrude(
 	}
 }
 
+void GeomUtils::createVoids(std::vector<carve::geom2d::P2> merged_path,
+	std::vector<carve::triangulate::tri_idx> triangulated,
+	std::vector<std::pair<size_t, size_t>> path_all_loops,
+	std::vector<std::vector<carve::geom2d::P2>>	face_loops,
+	std::stringstream& err)

```

Comment:
Either you need to pass by reference ( `std::vector<...>& name`) or you need to return the calculated values (preferable).

Reply:
Resolved:  71ec6b7